packages/react-dom/src/events/__tests__/ChangeEventPlugin-test.internal.js

```diff
@@ -122,78 +147,71 @@ describe('ChangeEventPlugin', () => {
     var called = 0;
 
     function cb(e) {
-      called += 1;
+      called++;
+      expect(e.type).toBe('change');
     }
 
-    var input = ReactTestUtils.renderIntoDocument(
+    var input = ReactDOM.render(
       <input type="radio" onChange={cb} />,
+      container,
     );
-    setUntrackedValue(input, true);
-    ReactTestUtils.SimulateNative.click(input);
-    ReactTestUtils.SimulateNative.click(input);
+
+    setUntrackedChecked.call(input, true);
+    input.dispatchEvent(new Event('click', {bubbles: true, cancelable: true}));
+    input.dispatchEvent(new Event('click', {bubbles: true, cancelable: true}));
     expect(called).toBe(1);
   });
 
   it('should deduplicate input value change events', () => {
-    var input;
     var called = 0;
 
     function cb(e) {
-      called += 1;
+      called++;
       expect(e.type).toBe('change');
     }
 
-    [
-      <input type="text" onChange={cb} />,
-      <input type="number" onChange={cb} />,
-      <input type="range" onChange={cb} />,
```

Comment:
Seems like it was important to do this for all types of inputs, and not just `'text`'?